frontends/web/src/decorators/translate.ts

```diff
@@ -32,14 +32,14 @@ interface InterpolationValues {
 /**
  * This interface describes the default value to return if no translation was found.
  */
-// interface DefaultValue {
-//     defaultValue: any;
-// }
+interface DefaultValue {
```

Comment:
What does this commit do, why is this needed?

Reply:
Ah, then please ignore (parts of) my comments before.